public/index.html

```diff
@@ -7,16 +7,7 @@
         <meta name="baseUriPath" content="::baseUriPath::" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="description" content="unleash" />
-
         <title>Unleash - Enterprise ready feature toggles</title>
-        <link
-            href="https://fonts.googleapis.com/icon?family=Material+Icons"
-            rel="stylesheet"
-        />
-        <link
-            href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700"
```

Comment:
We need to include the fonts for the different weight types as well (300, 400, 500, 700) and add the correct font weight mapping in the @font-face ruleset. Right our fonts have only the default regular weight regardless of the CSS settings.